### DISCUSSION TITLE
Total deconstruct functionality

### DISCUSSION THREAD

**Discussion by PiggyWhiskey** [notnotmelon/factorissimo-2-notnotmelon#97] [Ideas]
I use recursion a lot, and I was wondering if it's a possibility to fully strip down a building including all sub-buildings to allow upgrades/new layouts.

Perhaps a special storage building that when placed adjacent to a Factorissimo building that will auto deconstruct everything internal (from the deepest level first)

**Comment by notnotmelon** [comment]
One day it will be done
